scania-digital-design-system/sdds
# feat(stepper): webcomponent version of stepper

### NEEDS UPDATED Z-INDEX

**Describe pull-request**  
Created a webcomponent version for stepper.

**Solving issue**  
Fixes: [AB#2699](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2699)

**How to test**  
1. Go to storybook link below.
2. Check in Components -> Stepper -> Webcomponent

**Screenshots**  
-

**Additional context**  
-
